spec/factories/teachers.rb

```diff
@@ -0,0 +1,13 @@
+FactoryBot.define do
+  factory :teacher do
+    first_name do
+      Faker::Name.first_name
+    end
+    last_name do
+      Faker::Name.last_name
+    end
+    email { Faker::Internet.email }
+    password { 'malutka1' }
```

Comment:
Marta, please **never ever commit your credentials**. There are bots out there that frequently scan  github repositories for clean text passwords like this one and try to log in to various portals with hope of generating any profit to the author of the bot.

If you're using this password anywhere, I beg you to stop reading this comment and change this password everywhere. I advise using a password manager too, something like `bitwarden`. Please don't use LastPass tho, it just had a breach and it wasn't their first one.

Reply:
Check:)